sqlscripts/replication/0003-jaeger-index-local.sql

```diff
@@ -0,0 +1,14 @@
+CREATE TABLE IF NOT EXISTS %s ON CLUSTER '{cluster}'
```

Comment:
regarding the file name, we don't use the `-local` for the other local tables. We should probably unify naming.

Reply:
@pavolloffay I do, actually. For every local table creation file in replication folder name ends with ``-local``.